WorkspaceServer/Packaging/IPackage.cs

```diff
@@ -28,13 +27,17 @@ public interface IMightSupportBlazor : IPackage
     {
         bool CanSupportBlazor { get; }
     }
+    public interface ICreateWorkspace : IPackage
+    {
+        Task<Workspace> CreateRoslynWorkspaceAsync(Budget budget);
```

Comment:
What type of workspace will this interface create for run or for language services ?